etc/kayobe/kolla.yml

```diff
@@ -577,7 +577,7 @@ kolla_enable_central_logging: true
 #kolla_enable_designate:
 #kolla_enable_destroy_images:
 #kolla_enable_elasticsearch:
-kolla_enable_elasticsearch_curator: true
+kolla_enable_elasticsearch_curator: "{{ not os_distribution == 'rocky' }}"
```

Comment:
Surely this should be handled by kayobe/kolla-ansible?

Reply:
More like Kayobe, but we're explicitly enabling that - so what Kayobe/Kolla-Ansible should do - fail on prechecks?

Reply:
What about kolla_enable_elasticsearch_curator: "{{ kolla_enable_elasticsearch }}"? Is this acceptable?

Reply:
Actually, if enable_elasticsearch is false on rocky, then does it matter whether curator is enabled?

Reply:
There is no container image for elasticsearch-curator - so it won't pull.